rhoshift/utils/operator/cleanup.py

```
@@ -0,0 +1,29 @@
+        if not os.path.exists(script_path):
+            raise FileNotFoundError(f"Cleanup script not found at {script_path}")
+        
+        # Make the script executable
+        os.chmod(script_path, 0o755)
+        
+        # Execute the script
+        cmd = f"sh {script_path} "
+        run_command(cmd, live_output=True, max_retries=0)
+        
+    except Exception as e:
+        raise RuntimeError(f"Failed to execute cleanup script: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)